clients/keycloak/src/main/kotlin/DefaultKeycloakClient.kt

```diff
@@ -383,6 +383,51 @@ class DefaultKeycloakClient(
         }.getOrElse {
             throw KeycloakClientException("Failed to load client roles for user '${id.value}'.", it)
         }.body()
+
+    override suspend fun addUserToGroup(username: UserName, groupName: GroupName) {
+        // FIXME: Unfortunately, no way to distinguish if there was a technical problem or the user was not found.
```

Comment:
In this class, typically `runCatching` is used for exception handling. For reasons of consistency, it would make sense to use this here as well.

Reply:
I did simplify exception handling by letting KeycloakClientExceptions pass through the layers.